### ISSUE TITLE
Support for bulk transaction with localId

### ISSUE THREAD

**Issue by almir1982** [crnk-project/crnk-framework#560]
Hi,
I am trying to insert bulk structure in transaction (example, customer --> contact Map --> contacts) but my primary keys in table are generated as sequence id on DB so i can't provide them during POST invocation on resource. 

I tried using operations module with example,
`[{
		"op" : "POST",
		"path" : "customer",
		"value" : {
			"type" : "customer",
			"attributes" : {
				"shortDesc" : "Test"
			},
			"relationships" : {
				"customerContacts" : {
					"data" : [{
							"type" : "customerContact"
						}
					]
				}
			}
		}
	},
	{
		"op" : "POST",
		"path" : "customerContact",
		"value" : {
			"type" : "customerContact",
			"attributes" : {
				"roleType" : "Test"
			}
		}
	}
]`
but this is not passing because as I wrote above I still don't have id's because they will be generated after insert of records into tables.

Note:
I asked about this on crnk lobby and I was told to open issue because this is currently not supported by crnk library.

Thanks for the help
Almir

**Comment by remmeier** [comment]
what should be done:

- Resource.java complemented by a localId
- OperationsModule should honor localId dependencies when sorting requests
- OpterationsModule should replace localIds by real ids after every created resource in all dependencies to that resource.

**Comment by almir1982** [comment]
Yes this is what is needed to support bulk action in crnk library. But please respect that id's on resources sometimes are not String but Long. Because Hibernate is below exposed  DTO model this could make issue with queries.
